### ISSUE TITLE
Support for RFC 2136 - Dynamic Updates in the Domain Name System (DNS UPDATE)

### ISSUE THREAD

**Issue by Chaz6** [nickbabcock/dness#9]
A dynamic DNS client should be able to update DNS without using HTTP(S) or other out-of-band mechanmisms.

https://tools.ietf.org/html/rfc2136

**Comment by nickbabcock** [comment]
Agreed. DNS UPDATE should be supported. The issue is that out of the three registrars / providers I use, none of them support it 😢
